gridstatus/spp.py

```diff
@@ -1161,17 +1166,12 @@ def _get_marketplace_session(self) -> dict:
         """
         html = requests.get(FILE_BROWSER_API_URL)
```

Comment:
This request returns a 404, but it still has a `JSESSIONID` and `ZSRF-TOKEN` in the cookies of the response.